Makefile

```diff
@@ -83,7 +83,7 @@ fallthrough: submodules
 manifests:
 	@$(WARN) Deprecated. Please run make generate instead.
 
-generate: $(KUSTOMIZE) go.vendor go.generate gen-kustomize-crds
+generate: $(HELM) $(KUSTOMIZE) go.vendor go.generate gen-kustomize-crds
```

Comment:
Note that we still lint / build with helm2. This is due to the structure of the `build` submodule. Making changes there would cause significant churn and is not required to move to only supporting helm3 install.